source/frontend/src/components/Navbar.jsx

```diff
@@ -5,49 +5,63 @@ import PropTypes from 'prop-types'
 import Login from './login';
 import Logout from './logout';
 import { useGoogleDocs } from '../GoogleDocsContext';
+import placeholderImageUrl from './placeholder.png'; // Adjust the path if necessary
 
 
 
 
 function Navbar() {
     const { comments, accessToken } = useGoogleDocs();
     const [isUserLoggedIn, setIsUserLoggedIn] = useState(false);  // State to track login status
+    const [profileImageUrl, setProfileImageUrl] = useState(null);
+    const [userName, setUsername] = useState(null);
+    // const placeholderImageUrl = './placeholder.png';  // Placeholder image URL
+
 
 
     const onSuccessLogin = (res) => {
         console.log("LOGIN SUCCESS! Current user :", res.profileObj);
-        console.log("badaboom");
         setIsUserLoggedIn(true);  // Set login status to true on successful login
-
+        console.log(res.profileObj.imageUrl);
+        setProfileImageUrl(res.profileObj.imageUrl);
+        setUsername(res.profileObj.name);
     };
 
     const onSuccessLogout = (res) => {
         console.log("Log out successfull");
-        console.log("badaboom");
         setIsUserLoggedIn(false);  // Set login status to true on successful login
 
     };
-    return (
-        <div className='navbar p-8 border mb-12 flex justify-between items-center'> {/* Flexbox container */}
```

Comment:
this part is mentioned twice

Reply:
where do you see the other part @Jyotsnaiyer ? I only see it there

Reply:
Line 43 to line 50